pkg/apis/duck/v1beta1/annotations.go

```diff
@@ -19,6 +19,9 @@ package v1beta1
 import (
 	"context"
 	"fmt"
+	"github.com/google/go-cmp/cmp"
```

Comment:
can you create an issue to track that we shouldn't be duplicating all this constants nor the methods. And that we should move it outside the v1alpha1 and v1beta1 packages?

Reply:
https://github.com/google/knative-gcp/issues/1274